manifests/platform.yaml

```diff
@@ -86,6 +86,7 @@ spec:
       # cray-nexus
       - artifactory.algol60.net/csm-docker/stable/docker.io/sonatype/nexus3:3.68.1-1
       - artifactory.algol60.net/csm-docker/stable/cray-nexus-setup:0.11.1
+      - artifactory.algol60.net/csm-docker/stable/docker-kubectl:1.24.17
```

Comment:
I think this image is already there at line 79

Reply:
Yes, but I wasn't sure if this was a part documentation-type list or not.  Since `cray-nexus` requires `docker-kubectl`, I added it under the `# cray-nexus` heading.  

I think it's confusing to list it only once under some specific component when many use it.  Why isn't there a `# Common Images` heading or something where images like `docker-kubectl` are listed?

Reply:
Removed `docker-kubectl` from both `cray-nexus` and `cray-ceph-csi-rbd and cray-ceph-csi-cephfs` sections and put it in new `Common Images` section.